api/proto/core/pipeline/cron/cron.proto

```diff
@@ -47,6 +47,21 @@ service CronService {
       put: "/api/pipeline-crons/{cronID}",
     };
   }
+
+  rpc CronDisable (CronDisableRequest) returns (CronDisableResponse) {
+    option (google.api.http) = {
+      post: "/api/pipeline-crons/actions/disable",
```

Comment:
use restful style `PUT /api/pipeline-crons/{cronID}/actions/disable`.

and you need add opposite `enable` method.

Reply:
Within the disable method, cron is searched according to pipelinesource and pipelineymlname, and then set to disable and empty the expression

Reply:
not like cron start and stop to set enable and disable

Reply:
> Within the disable method, cron is searched according to pipelinesource and pipelineymlname, and then set to disable and empty the expression

you should not remove the expression when disable. just set `enable` field to false is enough.

Reply:
> not like cron start and stop to set enable and disable

what's the difference between start/stop and disable?  I think stop=disable

Reply:
Move the disabled API into the created method